deploy.sh

```diff
@@ -12,9 +12,9 @@ sudo zypper --non-interactive in --no-recommends \
   cockpit
 
 sudo systemctl start cockpit
-sudo systemctl start d-installer
+sudo systemctl start org.opensuse.DInstaller
```

Comment:
this looks strange, as I am not aware about any other systemd service that use such name.